retrieval_service/datastore/providers/cloudsql_postgres.py

```diff
@@ -330,17 +330,34 @@ async def get_amenity(self, id: int) -> Optional[models.Amenity]:
         return res
 
     async def amenities_search(
-        self, query_embedding: list[float], similarity_threshold: float, top_k: int
+        self,
+        query_embedding: list[float],
+        similarity_threshold: float,
+        top_k: int,
+        open_time: Optional[str],
+        open_day: Optional[str],
     ) -> list[models.Amenity]:
+
+        open_time_datetime = None
+        filter_query = "WHERE "
+        if open_time and open_day:
```

Comment:
What happens if they specified one and not the other?

Reply:
at the endpoint level it will throw an error. Within the providers, it will just not use the filter.